### PR TITLE
Refactor/drag disabled

### PR DESCRIPTION
**Pull request by karin-on**

### CODE REVIEW

**Comment by karin-on** [comment]
messed a bit, sorry. probably the best thing to do right now is to take care of styling branch and then rebase this branch (refactor/dragDisabled) onto master(?).

File: src/components/Game/Game.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ class GameView extends Component {
             letters: [],
             currentAnswer: [],
             isSubmitVisible: false,
-            dragDisabled: this.getDragDisabled(),
+            dragActive: this.getDragActive(),
```
Comment (tomecko) [inline src/components/Game/Game.js:27]: :clap:

File: src/components/Game/Game.js [hunk 2]
```diff
@@ -56,28 +56,21 @@ class GameView extends Component {
         window.removeEventListener('resize', this.debouncedResizeListener);
     }
 
-    getDragDisabled = () => isScreenSmall();
+    getDragActive = () => !isScreenSmall();
```
Comment (tomecko) [inline src/components/Game/Game.js:59]: :clap: :clap:

File: src/components/Game/Game.js [hunk 3]
```diff
@@ -266,25 +266,25 @@ class GameView extends Component {
                             <Letters
                                 letters={letters}
                                 clicked={this.handleLetterClick}
-                                dragDisabled={dragDisabled}
+                                dragDisabled={!dragActive}
```
Comment (tomecko) [inline src/components/Game/Game.js]: maybe rename the prop as well?

**Review by tomecko — APPROVED** [review]

**Comment by tomecko** [comment]
> messed a bit, sorry. probably the best thing to do right now is to take care of styling branch and then rebase this branch (refactor/dragDisabled) onto master(?).

right!
good article on https://git-scm.com/book/en/v2/Git-Branching-Rebasing

also `git reset COMMIT_HASH` (e.g. `git reset 6e21973`) is a good thing to know when you need to come back to certain commit (in case you don't know it)

**Review by tomecko — APPROVED** [review]

**Pull request merged**